_bibliography/pint.bib

```diff
@@ -951,6 +951,16 @@ @article{MadayEtAl2007
 	year = {2007},
 }
 
+@inbook{Ulbrich2007,
+	author = {Stefan Ulbrich},
+	title = {7. Generalized SQP Methods with ``Parareal'' Time-Domain Decomposition for Time-Dependent PDE-Constrained Optimization},
+	booktitle = {Real-Time PDE-Constrained Optimization},
+	chapter = {},
+	pages = {145-168},
+	doi = {10.1137/1.9780898718935.ch7},
```

Comment:
Now add `url = {https://dx.doi.org/10.1137/1.9780898718935.ch7},` and we are good to go :)